cmd/server/main.go

```diff
@@ -54,19 +56,29 @@ func main() {
     log.Println(`server terminated`)
 }
 
-func parseFlags(addr *string) {
-    flag.StringVar(addr, `a`, `localhost:8080`, `address and port to run server`)
-    flag.Parse()
-}
-
-func configureServer(addr string, appLog *log.Logger) *http.Server {
+func configureServer(addr string, appLog *log.Entry) *http.Server {
     appLog.Printf(`server bootstrap, address: %s`, addr)
-    metricRepo := repository.NewInMemoryRepo()
-    metricService := service.New(metricRepo, newLog("service"))
-    metricHandler := handler.New(metricService, newLog("handler"))
+
+    metricRepo, err := repository.NewFileRepo(
+        time.Duration(flagStoreInt),
+        flagStorePath,
+        flagRestore,
+        newLogger("repo"),
+    )
+    if err != nil {
+        appLog.Fatal(err)
```

Comment:
К ошибке лучше всегда добавлять какой-то контекст:

```suggestion
        appLog.Fatalf("не удалось создать хендлер файлового репозитория: %v", err)
```